src/main/java/io/appform/dropwizard/sharding/config/ShardingBundleOptions.java

```diff
@@ -11,4 +11,14 @@
 @AllArgsConstructor
 public class ShardingBundleOptions {
     private boolean skipReadOnlyTransaction = false;
+
+    @Builder.Default
+    private boolean encryptionSupport = false;
```

Comment:
maybe rename to `encryptionSupported` or `encryptionSupportEnabled`

Reply:
Addressed.